lang/java/reef-tang/tang/src/main/java/org/apache/reef/tang/formats/ConfigurationModule.java

```diff
@@ -143,8 +143,24 @@ public final ConfigurationModule set(final Param<? extends Number> opt, final Nu
     return c;
   }
 
+  public final <T> ConfigurationModule setMultipleValues(final Param<T> opt, final Iterable<String> values) {
+    ConfigurationModule c = deepCopy();
```

Comment:
You probably don't need the initial `deepCopy()`, as `set()` creates them anyhow.